app/src/main/java/tech/ula/utils/ServerUtility.kt

```diff
@@ -79,6 +79,23 @@ class ServerUtility(
         }
     }
 
+    private fun startXSDLServer(session: Session): Long {
```

Comment:
I think this function, and maybe the asset names could be changed to better reflect what they're doing. It doesn't seem like we're really starting a 'server' per se.

Reply:
I've changed it to `setDisplayNumberAndStartTwm` to more accurately describe what the startup script is doing.